pkg/cmd/admin/validate/validate.go

```diff
@@ -16,8 +16,9 @@ const (
 The commands here allow administrators to validate the integrity of configuration files.
 `
 
-	validateDeprecationMessage = `The commands here are deprecated and will be removed. Use 'openshift ex diagnostics' to run configuration validations instead.
-See sub-command help text for specific instructions with 'openshift ex diagnostics'.`
+	validateDeprecationMessage = `and will be removed.
```

Comment:
Looks like you missed the master and node variants.  Also, two lines was already pushing it, don't add a third.